examples/basics/scene/complex_image.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# vispy: gallery 30
+# -----------------------------------------------------------------------------
+# Copyright (c) Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+"""
+Simple use of SceneCanvas to display an Image.
+"""
```

Comment:
Could you move this to `examples/scene/` so it shows up in the gallery? You'd also have to format the docstring a little differently. For example:

https://github.com/vispy/vispy/blob/49b0c499238750d95806ab75a4706e2eb5c2a84a/examples/scene/colored_line.py#L1-L11

It is basically RST so a title and then any description you want below it.

Reply:
Also didn't this used to have key bindings to cycling through some of the settings?